Testing/CASLTests/Exceptions/StringNullOrEmptyExceptionTests.cs

```diff
@@ -21,31 +22,35 @@ public void Ctor_WhenInvokedWithNoParam_CorrectlySetsMessage()
         var exception = new StringNullOrEmptyException();
 
         // Assert
-        Assert.Equal("The string must not be null or empty.", exception.Message);
+        var expectedExceptionMessage = "The string must not be null or empty.";
+        exception.Message.Should().Be(expectedExceptionMessage);
     }
 
     [Fact]
     public void Ctor_WhenInvokedWithSingleMessageParam_CorrectlySetsMesage()
     {
         // Act
-        var exception = new StringNullOrEmptyException("test-message");
+        var exceptionMessage = "test-message";
+        var exception = new StringNullOrEmptyException(exceptionMessage);
 
         // Assert
-        Assert.Equal("test-message", exception.Message);
+        exception.Message.Should().Be(exceptionMessage);
     }
```

Comment:
Please refactor to the suggested change shown below.

**Reason:**  This will make the test follow the AAA (triple A) test layout.

```suggestion
    [Fact]
    public void Ctor_WhenInvokedWithSingleMessageParam_CorrectlySetsMesage()
    {
        // Arrange
        var expected = "test-message";
        
        // Act
        var exception = new StringNullOrEmptyException(expected);

        // Assert
        exception.Message.Should().Be(expected);
    }
```

Reply:
Changed test to follow AAA test layout.